lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -598,7 +607,7 @@ def add_normalized_threeds_2_data(xml, payment_method, options)
         xml.tag!('cavvAlgorithm', threeds_2_options[:cavv_algorithm]) if threeds_2_options[:cavv_algorithm]
         xml.tag!('paSpecificationVersion', threeds_2_options[:version]) if threeds_2_options[:version]
         xml.tag!('directoryServerTransactionID', threeds_2_options[:ds_transaction_id]) if threeds_2_options[:ds_transaction_id]
-        xml.tag!('commerceIndicator', options[:commerce_indicator]) if options[:commerce_indicator]
+        xml.tag!('commerceIndicator', ECI_BRAND_MAPPING[card_brand(payment_method).to_sym])
```

Comment:
_CyberSource: Pass commerce indicator if present_ #3350 made it possible for the caller to inject `options[:commerce_indicator]`and your PR removes that. Is it intentional? Would it make sense to give precedence to `options[:commerce_indicator]` if it is present?

Reply:
Thanks for catching that, I addressed this, could you review again please?